### ISSUE TITLE
Float values in settings are too precise (especially for people who don't know IEEE854)

### ISSUE THREAD

**Issue by hugo-vrijswijk** [microsoft/terminal#1398]
# Environment

```none
Windows build number: Microsoft Windows [Version 10.0.18362.175]
Windows Terminal version (if applicable): 0.2.1715.0

Any other software? No
```

# Steps to reproduce

1. Open settings
2. Set `acrylicOpacity` to `0.7`
3. Close Terminal
4. Open terminal again
5. Observe `acrylicOpacity` setting now being set to `0.69999998807907104`

# Expected behavior

`acrylicOpacity` should remain on `0.7`

# Actual behavior

`acrylicOpacity` has a floating point error.

**Comment by DHowett-MSFT** [comment]
*jsoncpp*'s default floating point precision is a little too ... precise. Probably something we can constrain.

**Comment by Willsatroyd** [comment]
This behaviour can be observed for any values other than 0.25, 0.5, 0.75 or 1.

**Comment by DHowett-MSFT** [comment]
Should we just switch these things to integral percent?

**Comment by zadjii-msft** [comment]
So this hasn't been _technically_ fixed. However, I believe it's been _practically_ fixed by #2475. Since we're no longer in the business of serializing the user's settings, then we won't be serializing floats incorrectly anymore.
